lib/explorer/series.ex

```diff
@@ -6118,6 +6118,36 @@ defmodule Explorer.Series do
     apply_series(series, :json_decode, [dtype])
   end
 
+  @doc """
+  Extracts a string series using a [`JSONPath`](https://goessner.net/articles/JsonPath/) from a series. 
```

Comment:
Let's rename this to `json_path` and use it throughout the docs:

```suggestion
  Extracts a string series using a [`json_path`](https://goessner.net/articles/JsonPath/) from a series. 
```